packages/documentation/src/sidebar.js

```diff
@@ -262,5 +232,33 @@ module.exports = {
         },
       ],
     },
+    {
+      id: 'documentation',
+      name: 'This site',
+      href: '/this-site',
```

Comment:
need feedback on this. It didn't really fit under the getting started section to me i.e. it's not something someone needs to read within their first month of working.

Reply:
I like having the separate section, but I don't really like it being called "this site." Maybe "Working on this site" or "Documentation guide"? "Updating documentation"? Not sure I've got a better name, I guess.

Reply:
This works for me. We can expand on this. 

![Screen Shot 2019-04-19 at 12 20 31](https://user-images.githubusercontent.com/4998130/56437693-9e7eac00-629d-11e9-8f3a-bd28029c59b3.png)